erts/doc/src/erlang.xml

```diff
@@ -1791,24 +1792,34 @@ example_fun(A1, A2) ->
     </func>
 
     <func>
-      <name name="error" arity="3" since="24.0"/>
+      <name name="error" arity="3" since="OTP 24.0"/>
       <fsummary>Raise an exception of class <c>error</c></fsummary>
       <desc>
         <p>Raises an exception of class <c>error</c> with the reason
-        <c><anno>Reason</anno></c>, where <c><anno>Reason</anno></c>
-        is any term. <c><anno>Args</anno></c> is expected to be the
-        list of arguments for the current function; it is used to
-        provide the arguments for the current function in the stack
-        back-trace.  If the <c>{error_info,
-        <anno>ErrorInfoMap</anno>}</c> option is given, it will be injected
-        into the stacktrace.</p>
-	<p>As evaluating this function causes an exception to be
-	raised, it has no return value.  The intent of the exception
-	class <c>error</c> is to signal that an unexpected error has
-	happened (for example, a function is called with a parameter
-	that has an incorrect type). See the guide about <seeguide
-	marker="system/reference_manual:errors">errors and error
-	handling</seeguide> for additional information.</p>
+          <c><anno>Reason</anno></c>, where <c><anno>Reason</anno></c>
+          is any term. <c><anno>Args</anno></c> is expected to be the
+          list of arguments for the current function; it is used
+          to provide the arguments for the current function in
+          the term <c>Where</c>. <c><anno>Args</anno></c> is expected to be the
+          list of arguments for the current function; it is used to
+          provide the arguments for the current function in the stack
+          back-trace. As evaluating this function causes an exception to be
+	  raised, it has no return value.</p>
+        <p>If the <c>error_info</c> option is given, the <c><anno>ErrorInfoMap</anno></c>
+          will be inserted into the stacktrace. The information given in the
+          <c><anno>ErrorInfoMap</anno></c> is used to be used by error formatters such as
+          <seemfa marker="stdlib:erl_error#format_exception/4"><c>erl_error</c></seemfa>
+          to provide more context around an error.</p>
+        <p>The default <c>module</c> of the <c><anno>ErrorInfoMap</anno></c> is the
+          module that the call to <seemfa marker="#error/3"><c>erlang:error/3</c></seemfa>
```

Comment:
```suggestion
module that the call to <c>error/3</c>
```

`error/3` can be called without the `erlang` prefix.

It should not be a link to itself.